coordinator/src/main/java/com/tencent/rss/coordinator/CoordinatorConf.java

```diff
@@ -66,7 +66,8 @@
   public static final ConfigOption<String> COORDINATOR_ACCESS_CHECKERS = ConfigOptions
       .key("rss.coordinator.access.checkers")
       .stringType()
-      .noDefaultValue()
+      .defaultValue("com.tencent.rss.coordinator.AccessCandidatesChecker,"
```

Comment:
user should config the checkers by themselves, or there will be unexpected actions for them

Reply:
this is the coordinator's configuration

Reply:
yes, user should config such feature by themselves including client, coordinator, shuffle server

Reply:
checkers should not be exposed to users, it's kind of administration parameter

Reply:
will only make load checker be default